src/newViews/ProjectEdit/GeoAreas/RegionCard/AddAdminLevel/index.tsx

```diff
@@ -40,8 +41,19 @@ function AddAdminLevel(props: Props) {
         activeRegion,
         adminLevels: adminLevelFromProps,
         isPublished,
+        onActiveAdminLevelChange,
     } = props;
 
+    /* FIXME when the parent component (ExpandableContainer) closes, this component is
+     *  unmounted. Then there is no way to remove activeAdminLevel. So we need to clear
+     *  the activeAdminLevel. Later we can add functionality to Expandable Container to
+     * take function which is called when the container is closed. Then we can remove
+     * the following code.
+     */
+    useEffect(() =>
+        () => onActiveAdminLevelChange(undefined),
+    [onActiveAdminLevelChange]);
```

Comment:
Why do we need to clear this value?

Reply:
@tnagorra when the expandable container is closed we need to remove the set activeTab value